packages/components/document-card/stories/document-card.stories.js

```diff
@@ -10,6 +10,12 @@ storiesOf('ts-document-card', module)
 		const selected = boolean('selected', true);
 		const description = text('description', 'CompanyName');
 		const mobileDescription = text('mobileDescription', 'Description');
+		const cardType = select('cardType', {
+			document: 'document',
```

Comment:
Please use list from the constants after adding them to there 👍

Reply:
ok